lib/memo_wise.rb

```diff
@@ -208,6 +245,9 @@ def #{method_name}#{args_str}
   #            calls to the same method with the same set of arguments will then
   #            return that saved result.
   #
+  #   Methods which implicitly or explicitly take block arguments cannot be
+  #   memoized.
```

Comment:
Let me know if you think I should add documentation around the proc workaround. I _think_ this is an unlikely case, and not worth documenting, but not a strong opinion.

Reply:
I agree; this is not worth documenting.